docs/API/configure-buffers.md

```diff
@@ -0,0 +1,115 @@
+# Configuration Guidelines
```

Comment:
I suggest renaming the document to `configuration-guidelines.md` instead of `configure-buffers.md` as it's going to be more general document in future rather than configuring buffers.